scons.py

```diff
@@ -3,6 +3,19 @@
 
 import sys
 import os
+import platform
+# Variables for storing required version of Python, and the version which is used to run this script.
+requiredPythonMajor ="2"
+requiredPythonMinor = "7"
+requiredPythonArchitecture = "32bit"
+installedPythonMajor = str(sys.version_info.major)
+installedPythonMinor = str(sys.version_info.minor)
+installedPythonArchitecture = platform.architecture()[0]
+# Ensure, that we are running with Python 2.7 32-bit, otherwise inform the user and exit.

```

Comment:
Please change this comment to be more generic (I.e. not mentioning the specific Python version) otherwise someone may forget to change the comment when the above variables are changed.